clients/drcachesim/tests/drmemtrace.threadsig.aarch64.raw/README

```diff
@@ -0,0 +1,50 @@
+How to re-generate the drmemtrace.threadsig.aarch64.raw files
+
+The application run is "threadsig" from https://github.com/DynamoRIO/drmemtrace_samples/blob/master/threadsig.aarch64/threadsig.cp
+
+To save repository space, build it with -Os and -static on an AArch64
```

Comment:
There is a binary present in the repo: https://github.com/DynamoRIO/drmemtrace_samples/blob/master/threadsig.aarch64/binaries/threadsig, which is same as the one checked in at dynamorio/clients/drcachesim/tests/drmemtrace.threadsig.aarch64.raw/threadsig. Is it ok to just use that one? Or should I recompile and regenerate the trace?

Reply:
Hmm, yeah, just running that one seems fine.  I will update the README to give that option.

Reply:
The output binary after following these steps is exactly the same.